src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -23,7 +23,7 @@
 class ChainDecoder implements DecoderInterface
 {
     protected $decoders = array();
-    protected $decoderByFormat = array();
+    private $decoderByFormat = array();
```

Comment:
Since `ChainDecoder` is not `final` it may be extended by user code which then in turn is able to access this property. Changing it to be `private` will break that code.